From 0dfe2a1825a2963b5e5b8adda98ecba4a59d30d7 Mon Sep 17 00:00:00 2001
From: Naresh Solanki <naresh.solanki@9elements.com>
Date: Tue, 31 Oct 2023 16:06:30 +0000
Subject: [PATCH 17/17] configuration: sbp1: Update threshold for BCM

Update threshold for BCM as:
BCM Warning limit: 5A, Critical limit 5.5A
BCM input voltage warning range: 385 - 395V
BCM input voltage critical range: 370 - 410V
BCM output voltage warning range: 11.68 - 12.33V
BCM output voltage critical range: 11.08 - 12.8V

Signed-off-by: Naresh Solanki <naresh.solanki@9elements.com>
Change-Id: I72833c0bad2261a277a9d5c20f9c45ca215dfe2f
---
 configurations/sbp1_psu.json | 246 +++++++++++++++++++++++++++++++++--
 1 file changed, 235 insertions(+), 11 deletions(-)

diff --git a/configurations/sbp1_psu.json b/configurations/sbp1_psu.json
index a1bee42..cfdf1a9 100644
--- a/configurations/sbp1_psu.json
+++ b/configurations/sbp1_psu.json
@@ -23,14 +23,42 @@
                         "Label": "vin2",
                         "Name": "upper critical",
                         "Severity": 1,
-                        "Value": 480.96
+                        "Value": 410
                     },
                     {
                         "Direction": "less than",
                         "Label": "vin2",
                         "Name": "lower critical",
                         "Severity": 1,
-                        "Value": 260
+                        "Value": 370
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "vin2",
+                        "Name": "upper non critical",
+                        "Severity": 0,
+                        "Value": 395
+                    },
+                    {
+                        "Direction": "less than",
+                        "Label": "vin2",
+                        "Name": "lower non critical",
+                        "Severity": 0,
+                        "Value": 385
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "iin2",
+                        "Name": "upper critical",
+                        "Severity": 1,
+                        "Value": 5.5
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "iin2",
+                        "Name": "upper non critical",
+                        "Severity": 0,
+                        "Value": 5
                     },
                     {
                         "Direction": "greater than",
@@ -45,6 +73,34 @@
                         "Name": "upper critical",
                         "Severity": 1,
                         "Value": 68
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "vout2",
+                        "Name": "upper critical",
+                        "Severity": 1,
+                        "Value": 12.8
+                    },
+                    {
+                        "Direction": "less than",
+                        "Label": "vout2",
+                        "Name": "lower critical",
+                        "Severity": 1,
+                        "Value": 11.08
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "vout2",
+                        "Name": "upper non critical",
+                        "Severity": 0,
+                        "Value": 12.33
+                    },
+                    {
+                        "Direction": "less than",
+                        "Label": "vout2",
+                        "Name": "lower non critical",
+                        "Severity": 0,
+                        "Value": 11.68
                     }
                 ],
                 "Type": "PLI1209BC"
@@ -82,21 +138,49 @@
                 "Name": "BCM1_TEMP",
                 "Name1": "BCM1",
                 "PollRate": 5.0,
-                "PowerState": "Always",
+                "PowerState": "On",
                 "Thresholds": [
                     {
                         "Direction": "greater than",
                         "Label": "vin2",
                         "Name": "upper critical",
                         "Severity": 1,
-                        "Value": 480.96
+                        "Value": 410
                     },
                     {
                         "Direction": "less than",
                         "Label": "vin2",
                         "Name": "lower critical",
                         "Severity": 1,
-                        "Value": 260
+                        "Value": 370
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "vin2",
+                        "Name": "upper non critical",
+                        "Severity": 0,
+                        "Value": 395
+                    },
+                    {
+                        "Direction": "less than",
+                        "Label": "vin2",
+                        "Name": "lower non critical",
+                        "Severity": 0,
+                        "Value": 385
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "iin2",
+                        "Name": "upper critical",
+                        "Severity": 1,
+                        "Value": 5.5
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "iin2",
+                        "Name": "upper non critical",
+                        "Severity": 0,
+                        "Value": 5
                     },
                     {
                         "Direction": "greater than",
@@ -111,6 +195,34 @@
                         "Name": "upper critical",
                         "Severity": 1,
                         "Value": 68
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "vout2",
+                        "Name": "upper critical",
+                        "Severity": 1,
+                        "Value": 12.8
+                    },
+                    {
+                        "Direction": "less than",
+                        "Label": "vout2",
+                        "Name": "lower critical",
+                        "Severity": 1,
+                        "Value": 11.08
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "vout2",
+                        "Name": "upper non critical",
+                        "Severity": 0,
+                        "Value": 12.33
+                    },
+                    {
+                        "Direction": "less than",
+                        "Label": "vout2",
+                        "Name": "lower non critical",
+                        "Severity": 0,
+                        "Value": 11.68
                     }
                 ],
                 "Type": "PLI1209BC"
@@ -148,21 +260,49 @@
                 "Name": "BCM2_TEMP",
                 "Name1": "BCM2",
                 "PollRate": 5.0,
-                "PowerState": "Always",
+                "PowerState": "On",
                 "Thresholds": [
                     {
                         "Direction": "greater than",
                         "Label": "vin2",
                         "Name": "upper critical",
                         "Severity": 1,
-                        "Value": 480.96
+                        "Value": 410
                     },
                     {
                         "Direction": "less than",
                         "Label": "vin2",
                         "Name": "lower critical",
                         "Severity": 1,
-                        "Value": 260
+                        "Value": 370
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "vin2",
+                        "Name": "upper non critical",
+                        "Severity": 0,
+                        "Value": 395
+                    },
+                    {
+                        "Direction": "less than",
+                        "Label": "vin2",
+                        "Name": "lower non critical",
+                        "Severity": 0,
+                        "Value": 385
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "iin2",
+                        "Name": "upper critical",
+                        "Severity": 1,
+                        "Value": 5.5
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "iin2",
+                        "Name": "upper non critical",
+                        "Severity": 0,
+                        "Value": 5
                     },
                     {
                         "Direction": "greater than",
@@ -177,6 +317,34 @@
                         "Name": "upper critical",
                         "Severity": 1,
                         "Value": 68
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "vout2",
+                        "Name": "upper critical",
+                        "Severity": 1,
+                        "Value": 12.8
+                    },
+                    {
+                        "Direction": "less than",
+                        "Label": "vout2",
+                        "Name": "lower critical",
+                        "Severity": 1,
+                        "Value": 11.08
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "vout2",
+                        "Name": "upper non critical",
+                        "Severity": 0,
+                        "Value": 12.33
+                    },
+                    {
+                        "Direction": "less than",
+                        "Label": "vout2",
+                        "Name": "lower non critical",
+                        "Severity": 0,
+                        "Value": 11.68
                     }
                 ],
                 "Type": "PLI1209BC"
@@ -214,21 +382,49 @@
                 "Name": "BCM3_TEMP",
                 "Name1": "BCM3",
                 "PollRate": 5.0,
-                "PowerState": "Always",
+                "PowerState": "On",
                 "Thresholds": [
                     {
                         "Direction": "greater than",
                         "Label": "vin2",
                         "Name": "upper critical",
                         "Severity": 1,
-                        "Value": 480.96
+                        "Value": 410
                     },
                     {
                         "Direction": "less than",
                         "Label": "vin2",
                         "Name": "lower critical",
                         "Severity": 1,
-                        "Value": 260
+                        "Value": 370
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "vin2",
+                        "Name": "upper non critical",
+                        "Severity": 0,
+                        "Value": 395
+                    },
+                    {
+                        "Direction": "less than",
+                        "Label": "vin2",
+                        "Name": "lower non critical",
+                        "Severity": 0,
+                        "Value": 385
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "iin2",
+                        "Name": "upper critical",
+                        "Severity": 1,
+                        "Value": 5.5
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "iin2",
+                        "Name": "upper non critical",
+                        "Severity": 0,
+                        "Value": 5
                     },
                     {
                         "Direction": "greater than",
@@ -243,6 +439,34 @@
                         "Name": "upper critical",
                         "Severity": 1,
                         "Value": 68
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "vout2",
+                        "Name": "upper critical",
+                        "Severity": 1,
+                        "Value": 12.8
+                    },
+                    {
+                        "Direction": "less than",
+                        "Label": "vout2",
+                        "Name": "lower critical",
+                        "Severity": 1,
+                        "Value": 11.08
+                    },
+                    {
+                        "Direction": "greater than",
+                        "Label": "vout2",
+                        "Name": "upper non critical",
+                        "Severity": 0,
+                        "Value": 12.33
+                    },
+                    {
+                        "Direction": "less than",
+                        "Label": "vout2",
+                        "Name": "lower non critical",
+                        "Severity": 0,
+                        "Value": 11.68
                     }
                 ],
                 "Type": "PLI1209BC"
-- 
2.41.0

